### PR TITLE
fix location of report_data_select attributes

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a pull request! Maybe @nickanderson can review this?

File: enterprise-cfengine-guide/monitoring.markdown [hunk 1]
```diff
@@ -26,4 +26,4 @@ Clicking on an individual graph allows to select different time spans for which
 If you don't see any data, make sure that:
 
 * `cf-monitord` is running on your hosts. This is configurable through the lists `agents_to_be_enabled` and `agents_to_be_disabled` in `masterfiles/update/update_processes.cf`.
-* `cf-hub` has access to collecting the monitoring data from your hosts. This is configurable through the attributes in `report_data_select` in `masterfiles/controls/cf_serverd.cf`.
+* `cf-hub` has access to collecting the monitoring data from your hosts. This is configurable through the attributes in `report_data_select` in `masterfiles/controls/reports.cf`.
```
Comment (craigcomstock) [inline enterprise-cfengine-guide/monitoring.markdown]: maybe we need a reference to default_data_select_host_monitoring_include and default_data_select_policy_hub_monitoring_include to put into def.json?

Comment (nickanderson) [reply enterprise-cfengine-guide/monitoring.markdown]: You link to the page title and anchor

So something like [Anchor text][Masterfiles Policy Framework#configure-enterprise-measurement-monitoring-collection]

**Comment by nickanderson** [comment]
Closing, I took care of it.

**Pull request closed without merging**